src/cmd/mod.rs

```diff
@@ -88,6 +90,7 @@ pub enum ApiErrorKind {
     NoLongerSupported {},
     NoMatchingExit {},
     RateLimitExceeded {},
+    SaleNotFound {},
```

Comment:
@FlorianUekermann I added an error code for if the sale can't be applied. PTAL.

I was considering using a specific `PreconditionFailed` but it seems simpler to be very specific.

Reply:
:+1: